spec/controllers/api/users_controller_spec.rb

```diff
@@ -23,7 +24,7 @@
     end
 
     it 'should have properties for user 1' do
-      user = result.detect { |u| u['id'] == user_1.id }
+      user = result.detect { |u| u['id'].to_i == user_1.id }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.